docs/configuration/indexers.md

```diff
@@ -21,6 +21,8 @@ If you want more indexers added please either create a PR for it, post a comment
 
 import Indexers from '/snippets/indexers.mdx';
 
+import ipapproval from '/snippets/ipapproval.mdx';
```

Comment:
You need to use the component somewhere, and to be a valid component change to capitalized so `IpApproval` and then you can use it like `<IpApproval/>`. 

I would probably place it at the bottom of the **Setup** section, just above **Custom indexer definitions** 👍

Reply:
Cheers. I don't code, but I've spent enough time around FOOS that I can push the odd changes here and there.

I probably should have asked this initially, but is this the preferred method of adding these tips? I didn't examine the entire code, only the first example I found where a TIP was used?